verl/workers/actor/dp_actor.py

```diff
@@ -98,14 +98,20 @@ def _forward_micro_batch(
         response_length = micro_batch["responses"].size(-1)
         multi_modal_inputs = {}
         if "multi_modal_inputs" in micro_batch.keys():
-            if "image_bound" in micro_batch["multi_modal_inputs"][0]:  # minicpm-o logic
-                for key in micro_batch["multi_modal_inputs"][0].keys():
-                    multi_modal_inputs[key] = [inputs[key] for inputs in micro_batch["multi_modal_inputs"]]
-            else:
-                for key in micro_batch["multi_modal_inputs"][0].keys():
-                    multi_modal_inputs[key] = torch.cat(
-                        [inputs[key] for inputs in micro_batch["multi_modal_inputs"]], dim=0
-                    )
+            multi_modal_inputs_collected = {}
```

Comment:
Can we write a unified function that extracts the input to what should be passed in to the model?

Reply:
And inside the function, we can perform model-specific processing

Reply:
Done, thanks for reviewing!